test/parallel/test-http2-respond-file-fd.js

```diff
@@ -13,7 +14,8 @@ const {
   HTTP2_HEADER_CONTENT_LENGTH
 } = http2.constants;
 
-const fname = path.resolve(common.fixturesDir, 'elipses.txt');
+const fname = path.resolve(fixtures.path(), 'elipses.txt');
```

Comment:
This could be using method `fixtures.path`

Reply:
you mean instead of path.resolve? fixtures.path uses path.join in the background rather than resolve.

Reply:
const fname = fixtures.path('elipses.txt');